src/main/resources/locale_pt_PT.properties

```
@@ -0,0 +1,196 @@
+command.thankwords.sub.check.result=Donatador: %DONATOR%%NReceptor: %RECEIVER%
+command.thankwords.sub.list=Listar todos os padrões
+command.thankwords.sub.list.list=Following Pattern are active:
+command.thankwords.sub.loadDefault=Load default thankswords for a language.
+command.thankwords.sub.loadDefault.added=Added the following thankwords:
+command.thankwords.sub.loadDefault.available=Línguas disponíveis:
+command.thankwords.sub.remove=Delete a pattern
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
+conversation.cancel=cancelar
+conversation.canceled=Conversation canceled.
+conversation.exit=saída
+conversation.finished=Conversation finished.
+conversation.inProgress=Já está em curso uma conversação neste canal. Pode sair desta conversa digitando $conversation.exit$ ou $conversation.cancel$.
+conversation.start=Iniciando uma nova conversação.\n Você pode deixar a conversa a qualquer momento digitando $conversation.exit$ ou $conversation.cancel$.
+error.commandNotFound=Command não encontrado.
+error.invalidArguments= Argumentos inválidos
+error.invalidChannel=Invalid Channel
+error.invalidMessage=Mensagem inválida
+error.invalidNumber=Número inválido
+error.invalidRegex=Invalid regex pattern.
+error.invalidRole=Invalid Role
+error.missingPermission=Sinto falta da permissão %PERM% para funcionar correctamente.
+error.missingPermissionChannel= Isto deve-se à definição de permissões de anulação no canal %CHANNEL%.
+error.missingPermissionGuild=É devido a definições de permissão globais.
+error.noValidChannel=Não são fornecidos canais válidos.
+error.notABoolean=%INPUT% is not a boolean (true|false).
+error.notANumber=%INPUT% não é um número.
+error.notYourEmbed=Este não é o seu embutido!
+error.prefixTooLong=Prefix pode ter apenas 3 caracteres.
+error.roleAccess=Não pode interagir com role %ROLE%. Por favor, ajuste a hierarquia.
+error.userNotFound=Não conseguiria encontrar este utilizador.
+listener.messages.request.descr=Hey, acabou de agradecer a alguém, quer agradecer a alguém em particular? Se sim, clique no botão abaixo.
+listener.messages.request.descrThank=Obrigado por dar a alguém Reputação. Pode dar %MAIS% mais reputação.
+listener.messages.request.title=Posição de agradecimento
+listener.reaction.confirmation=%DONOR% deu reputação a %RECEIVER%.
+listener.reaction.removal=Reputação dada por %DONOR% foi removida.
+message.welcome=Perdão por me ter convidado.\n Você pode iniciar uma configuração fácil com o comando /setup.
+messages.destruction= Esta mensagem autodestruir-se-á em %MIN% minuto.
+thankType.answer=Resposta
+thankType.fuzzy=Fuzzy
+thankType.mention=Menção
+thankType.reaction=Reacção
+word.continue=Continuar
+word.done=Done
+word.skip=Skip
+words.code=Código
+words.jumpMarker=[%TARGET%](%URL%)
+words.language=Langauge
+words.level=Nível
+words.message=Mensagem
+words.name=Nome
+words.refMessage=Mensagem de referência
+words.reputation=Reputação
+words.role=Role
```

Looks untranslated

DeepL suggestion:
```suggestion
words.role=Papel
```